packages/sprotty/src/features/export/export.ts

```diff
@@ -41,14 +41,16 @@ export class ExportSvgKeyListener extends KeyListener {
 
 export interface RequestExportSvgAction extends RequestAction<ExportSvgAction> {
     kind: typeof RequestExportSvgAction.KIND
+    skipCopyStyles?: boolean
 }
 export namespace RequestExportSvgAction {
     export const KIND = 'requestExportSvg';
 
-    export function create(): RequestExportSvgAction {
+    export function create(skipCopyStyles?: boolean): RequestExportSvgAction {
```

Comment:
I would wrap this in an options object because
- passing booleans to a function is not the best API choice in general (`create(true)` can't be understood without looking into the function definition),
- when adding more options to an object, the order doesn't matter and you don't have to pass `undefined` for those that you don't need.